src/main/java/at/hannibal2/skyhanni/config/features/Misc.java

```diff
@@ -246,39 +246,63 @@ public static class DiscordRPC {
         @Expose
         @ConfigOption(name = "First Line", desc = "Decide what to show in the first line.")
         @ConfigEditorDropdown(values = {
-                "Nothing",
-                "Location",
-                "Purse",
-                "Bits",
-                "Stats",
-                "Held Item",
-                "Skyblock Date",
-                "Profile (Fruit)",
-                "Slayer",
-                "Custom"
+            "Nothing",
+            "Location",
+            "Purse",
+            "Bits",
+            "Stats",
+            "Held Item",
+            "Skyblock Date",
+            "Profile (Fruit)",
+            "Slayer",
+            "Custom",
+            "Auto",
+            "Garden Crop Milestone",
+            "Current Pet"
         })
         public Property<Integer> firstLine = Property.of(0);
 
         @Expose
         @ConfigOption(name = "Second Line", desc = "Decide what to show in the second line.")
         @ConfigEditorDropdown(values = {
-                "Nothing",
-                "Location",
-                "Purse",
-                "Bits",
-                "Stats",
-                "Held Item",
-                "Skyblock Date",
-                "Profile (Fruit)",
-                "Slayer",
-                "Custom"
+            "Nothing",
+            "Location",
+            "Purse",
+            "Bits",
+            "Stats",
+            "Held Item",
+            "Skyblock Date",
+            "Profile (Fruit)",
+            "Slayer",
+            "Custom",
+            "Auto",
+            "Garden Crop Milestone",
+            "Current Pet"
         })
         public Property<Integer> secondLine = Property.of(0);
 
         @Expose
         @ConfigOption(name = "Custom", desc = "What should be displayed if you select \"Custom\" above.")
         @ConfigEditorText
         public Property<String> customText = Property.of("");
+
+        @Expose
+        @ConfigOption(name = "Auto", desc = "What should be displayed when not in Garden or doing Slayers.")
```

Comment:
The description is a bit unclear in my opinion. It shows garden data or slayer data if enabled, right?